src/tests/WebApi.OpenAPI.SystemTest/Extensions/ApiExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace WebApi.OpenAPI.SystemTest.Extensions;
+
+public static class ApiExtensions
+{
+    public static void ConfigureControllers(this IServiceCollection services)
+    {
+        services.AddControllers();
+        services.ConfigureJsonSerializerOptions();
+    }
+
+    // SwashBuckle and TypedResults require different JsonOptions to be configured.
+    // https://github.com/domaindrivendev/Swashbuckle.AspNetCore/issues/2293
+    private static IServiceCollection ConfigureJsonSerializerOptions(this IServiceCollection services)
+    {
+        services.Configure<Microsoft.AspNetCore.Http.Json.JsonOptions>(options => ConfigureJsonOptions(options.SerializerOptions));
+        services.Configure<Microsoft.AspNetCore.Mvc.JsonOptions>(options => ConfigureJsonOptions(options.JsonSerializerOptions));
+
+        return services;
+    }
+
+    private static void ConfigureJsonOptions(JsonSerializerOptions options)
+    {
+        options.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
+        options.DictionaryKeyPolicy = JsonNamingPolicy.CamelCase;
+        options.WriteIndented = true;
```

Comment:
I know this is a test, but we should never return indented JSON, it will only consume more bandwith when the response is uncompressed

Reply:
If devs wants to see good-looking, indented JSON in their browser they should then install an extension for that